discord/colour.py

```diff
@@ -332,6 +332,24 @@ def nitro_pink(cls: Type[CT]) -> CT:
         .. versionadded:: 2.0
         """
         return cls(0xf47fff)
+    
+    @classmethod
+    def invis(theme: str = "dark", cls, Type: [CT]) -> CT:
```

Comment:
In my opinion, a better name for the method would be `embed_invisible`, since invis is not descriptive enough.